src/core/ext/census/grpc_plugin.c

```diff
@@ -31,11 +31,12 @@
  *
  */
 
-#include <limits.h>
 #include <string.h>
 
 #include <grpc/census.h>
 
+#include <limits.h>
```

Comment:
This line ends up being triggered otherwise: http://git.2f30.org/fortify-headers/file/include/stdlib.h.html#l42

I don't like this part of the patch, but don't know C well enough to find the root cause of why PATH_MAX isn't set.